rp2040-hal/Cargo.toml

```diff
@@ -7,6 +7,10 @@ homepage = "https://github.com/rp-rs/rp-hal"
 description = "A Rust Embeded-HAL impl for the rp2040 microcontroller"
 license = "MIT OR Apache-2.0"
 
+[package.metadata.docs.rs]
+features = ["rt"]
```

Comment:
Would it be worth adding `rtic-monotonic` and `defmt` to have the extra types and derive show in the doc too?